setup.py

```diff
@@ -279,19 +279,16 @@ def run(self):
     'cligj>=0.5',
     'click-plugins>=1.0',
     'six>=1.7',
-    'munch']
-
-if sys.version_info < (2, 7):
-    requirements.append('argparse')
-    requirements.append('ordereddict')
-
-if sys.version_info < (3, 4):
-    requirements.append('enum34')
+    'munch',
+    'argparse; python_version <= "3.4"',
+    'ordereddict; python_version <= "2.7"',
```

Comment:
Should ordereddict be < 2.7 and not <= 2.7? https://pypi.org/project/ordereddict/
Python 2.7.0 was released July 3, 2010, ordered dict 1.1 Januar 14, 2010. 

There was previously a discrepancy between setup.py ( < 2.7) and requirements.txt (<= 2.7). Now both is <= 2.7.